samples/modules/pmci/mctp/usb_endpoint/README.rst

```diff
@@ -0,0 +1,113 @@
+.. zephyr:code-sample:: mctp-usb-endpoint
+   :name: MCTP USB Endpoint Node Sample
+   :relevant-api: usbd_api
+
+   Create an MCTP endpoint node using the USB device interface.
+
+Overview
+********
+Sets up an MCTP endpoint node that listens for messages from the MCTP bus owner with EID 20.
+Responds with "Hello, bus owner" message.
+
+Requirements
+************
+A board and SoC that provide support for USB device capability (either FS or HS). Testing requires
+a USB host that has the ability to enumerate the MCTP interface and interact with the board using
+the MCTP protocol. An easy way to do this is a Python script.
+
+An example script to test this interface is provided below.
+
+.. code-block:: python
```

Comment:
Honestly probably better to put this into scripts or into the sample directory itself as a python script with requirements.txt and all rather than pushing users to copy/paste, and figure out how to get the python deps installed.

Can always include whole code files into the docs like this with [literalinclude](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude)